resources/assets/less/bem/beatmap-discussion-post.less

```diff
@@ -73,9 +73,11 @@
     }
 
     &--cancel {
+      .reset-input();
       position: absolute;
       top: 10px;
       right: 10px;
+      padding: 3px 13px; // how big the safari button would have been.
```

Comment:
probably better set width to `@beatmap-discussion--reply-padding` and add left/right padding accordingly?